libs/ui-toolkit/src/components/dialog/dialog.tsx

```diff
@@ -50,15 +52,20 @@ export function Dialog({
               className="focus:outline-none focus-visible:outline-none"
             />
           </DialogPrimitives.Close>
-          {title && (
-            <h1
-              className={`text-h5 text-black-95 dark:text-white-95 mt-0 mb-20 ${titleClassNames}`}
-              data-testid="dialog-title"
-            >
-              {title}
-            </h1>
-          )}
-          {children}
+          <div className="flex gap-12 max-w-full">
+            {icon && <div className="pt-8 fill-current">{icon}</div>}
+            <div data-testid="dialog-content" className="flex-1">
+              {title && (
+                <h1
+                  className={`text-h4 font-bold text-black-95 dark:text-white-95 mt-0 mb-6 capitalize ${titleClassNames}`}
```

Comment:
```suggestion
                  className={`text-h4 font-bold text-black-95 dark:text-white-95 mt-0 mb-6  ${titleClassNames}`}
```

Reply:
all dialogs have now capitalized letters, and Pete will ask me to change that as it is not consistent with the rest of the console style 

<img width="821" alt="Screenshot 2022-07-23 at 10 13 04" src="https://user-images.githubusercontent.com/16125548/180596865-22216e02-e906-4f19-a56c-c324994b5847.png">


<img width="1207" alt="Screenshot 2022-07-23 at 10 12 56" src="https://user-images.githubusercontent.com/16125548/180596854-fd385a12-2f45-4b5e-84e6-af51d253f775.png">